### ISSUE TITLE
[v1.0.0-beta] Tabs ugly behavior in case of page scroll appearing with one tab

### ISSUE THREAD

**Issue by MicheleLucini** [Dogfalo/materialize#5868]
## Expected Behavior
Tabs indicator width should be recalculated to match the new tab width after the divs is shown.

## Current Behavior
Tab indicator remain the same making the tabs scrollable.

## Steps to Reproduce (for bugs)
[Codepen example](https://codepen.io/Aozora/pen/pVNNer)

**Comment by acburst** [comment]
I'm not sure I'm seeing the problem. In your codepen, the 2 tabs are the same width

**Comment by MicheleLucini** [comment]
Sorry, i'm talking about the Little line that highlight the active tab

**Comment by MicheleLucini** [comment]
When the div shown by a tab is longer and require the body to scroll the width of the ul.tabs change but the Little line doesnt making the tabs scrollable

**Comment by MicheleLucini** [comment]
Try my codepen in a medium size screen

**Comment by MicheleLucini** [comment]
I removed the min-width to the div .maincontent in my pen, now you can see it at all window sizes on a desktop.

**Comment by MaximBalaganskiy** [comment]
the code taken from https://github.com/Dogfalo/materialize/issues/5058#issuecomment-371166237 fixes the issue
```
M.Tabs.prototype._calcRightPos = function(el) {
			return Math.ceil(this.el.clientWidth - el.position().left - el[0].getBoundingClientRect().width);
		};
```

**Comment by acburst** [comment]
Fixed in 2103d82f
